[CI-6201] Add ai onboarding to getting started page (#51)
bitrise-io/devcenter-new: docs/bitrise-platform/getting-started/signing-up-for-bitrise.mdx

import GlossTerm from '@site/src/components/GlossTerm';
import Partial_NOTESigningUpWithEmail from '@site/src/partials/note-signing-up-with-email.mdx';

> :::tip[Onboarding with an AI agent?]

Reviewer: ```suggestion
:::tip[Onboarding with an AI agent]
```

> **AI agents and coding assistants:** you don't need this browser sign-up. You can create the user's Bitrise account and connect everything **directly from your session** — the user just provides an email and a one-time verification code, no website visit required. Follow the **[Bitrise for AI agents onboarding runbook](/en/bitrise-platform/ai/onboarding-for-agents)** instead of the steps on this page.

Reviewer: ```suggestion
AI agents and coding assistants can create the user's Bitrise account and connect everything directly from their session. The user just provides an email and a one-time verification code, no website visit required. 

Follow the [Bitrise for AI agents onboarding runbook](/en/bitrise-platform/ai/onboarding-for-agents) instead of this guide.
```
